R/email_summary.R

```diff
@@ -23,6 +23,37 @@ get_ny_tz <- function() {
   ifelse(get_os() == "mac", "America/New_york", "America/New_York")
 }
 
+#' Retrieve Email for Active Content Lead
+#'
+#' `get_content_lead()` retrieves the email of teh currently assigned content
+#' lead. The function retrieves a list of content leads and their assignment
+#' dates, checks these against the current date, and returns a character string
+#' containing an email address.
+#' @param google_drive_auth Authorization token for the SER Google Drive account.
+#' @export
+get_content_lead <- function(google_drive_auth = drive_auth_token()) {
+  require(lubridate)
```

Comment:
You should never use `library()` or `require()` in R package code.

Instead, if you need to call functions without pkg::, use `@importFrom`. Here, you need to add this to the roxygen: `@importFrom lubridate today %within% interval mdy`. 

However, my preference here is that you limit the import to `importFrom lubridate %within%` and use `lubridate::` for the rest.